src/org/ticket/view/CreateTicket.java

```diff
@@ -35,6 +72,49 @@ public CreateTicket() {
 
     @Override
     public void actionPerformed(ActionEvent e) {
+        Object source = e.getSource();
+
+        if(source.equals(bcreate)) {
+            disable();
+
+            TicketController tc = new TicketController();
+
+            Ticket.TicketPriority tp = Ticket.TicketPriority.LOW;
+            String tps = cbpriority.getSelectedItem().toString();
```

Comment:
It is easier and I think even faster - though not apparent - to use `getSelectedIndex()`.